velox/functions/sparksql/String.h

```diff
@@ -1431,4 +1431,37 @@ struct LevenshteinDistanceFunction {
   }
 };
 
+template <typename T>
+struct Empty2NullFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  // Results refer to strings in the first argument.
+  static constexpr int32_t reuse_strings_from_arg = 0;
+
+  static constexpr bool is_default_ascii_behavior = true;
+
+  FOLLY_ALWAYS_INLINE bool call(
+      out_type<Varchar>& result,
+      const arg_type<Varchar>& input) {
+    return doCall(result, input);
+  }
+
+  FOLLY_ALWAYS_INLINE bool callAscii(
+      out_type<Varchar>& result,
+      const arg_type<Varchar>& input) {
```

Comment:
This fast path could be removed as ascii characters are not handled specially.

Reply:
I'm not sure the details of current ascii fast path implementation, you mean I dont need set `is_default_ascii_behavior=true` and provide `callAscii` function, Velox could go ascii fast path and invoke `call` by default?

Reply:
done